cypress/integration/pages/onDemandRadioEpisode/tests.js

```diff
@@ -0,0 +1,64 @@
+// For testing important features that differ between services, e.g. Timestamps.
+// We recommend using inline conditional logic to limit tests to services which differ.
+export const testsThatAlwaysRun = ({ service, pageType }) => {
+  describe(`No testsToAlwaysRun to run for ${service} ${pageType}`, () => {});
+};
+
+// For testing feastures that may differ across services but share a common logic e.g. translated strings.
+export const testsThatFollowSmokeTestConfig = ({ service, pageType }) =>
+  describe(`Tests for ${service} ${pageType}`, () => {
+    describe('On Demand Radio body', () => {
+      it('should render a H1, which contains/displays a styled headline', () => {
```

Comment:
We can remove this test, as it has been implemented by an integration test instead: https://github.com/bbc/simorgh/blob/22d01e907c47c6b6180464a722099dc991420f0f/src/integration/pages/onDemandRadioEpisode/commonEpisodeTests.js#L2

Reply:
Removed